packages/offix-datastore/src/storage/adapters/IndexedDBStorage.ts

```diff
@@ -1,16 +1,15 @@
-import { Model, PersistedModel } from "../../models";
 import { IStorageAdapter } from "../Storage";
-import { getStoreNameFromModelName } from "../core";
 import { PredicateFunction } from "../../predicates";
 
-const DB_NAME = "offix-datastore";
-
+/**
+ * Web Storage Implementation for DataStore using IndexedDB
```

Comment:
[Offtopic] We will need 2 storages initiatly. Web storage and SQLLite storage. I think it is worth to create issue for ReactNative support with SQLLite storage - it might be that default store will not support this predicates and we will need some direct access

Reply:
I was thinking about this last night too. What is our fallback for when someone is using a browser in private mode, i.e. FireFox, that doesn't support IndexedDB?
@wtrocki @Eunovo